openapi/components/schemas/users/update-user-payload.json

```diff
@@ -0,0 +1,115 @@
+{
+  "UpdateUserRequestModel": {
+    "type": "object",
+    "properties": {
+      "phoneNumbers": {
+        "type": "array",
+        "items": {
+          "type": "string",
+          "pattern": "^[+]?[0-9]{1,3}?[0-9]{10}$"
+        }
+      },
+      "name": {
+        "type": "string"
+      },
+      "bloodGroup": {
+        "type": "string",
+        "enum": [
+          "A+",
+          "A-",
+          "B+",
+          "B-",
+          "AB+",
+          "AB-",
+          "O+",
+          "O-"
+        ]
+      },
+      "lastDonationDate": {
+        "type": "string",
+        "format": "date-time"
+      },
+      "height": {
+        "type": "number"
+      },
+      "weight": {
+        "type": "number"
+      },
+      "gender": {
+        "type": "string",
+        "enum": [
+          "Male",
+          "Female",
+          "Other"
+        ]
+      },
+      "dateOfBirth": {
+        "type": "string",
+        "format": "date"
+      },
+      "age": {
+        "type": "number"
+      },
+      "area": {
+        "type": "string"
+      },
+      "city": {
+        "type": "string"
+      },
+      "latitude": {
+        "type": "number",
+        "minimum": -90,
+        "maximum": 90
+      },
+      "longitude": {
+        "type": "number",
+        "minimum": -180,
+        "maximum": 180
+      },
+      "geohash": {
+        "type": "string"
+      },
+      "donationStatus": {
+        "type": "string",
+        "enum": [
+          "yes",
+          "no"
+        ]
+      },
+      "NIDFront": {
+        "type": "string"
+      },
+      "NIDBack": {
+        "type": "string"
+      },
+      "lastVaccinatedDate": {
```

Comment:
What kind of vaccination?

Reply:
this is the date for latest vaccination of any kind.